CMakeLists.txt

```diff
@@ -321,6 +322,34 @@ endif()
 
 # TODO: Check if ADIOS2 is parallel when openPMD_HAVE_MPI is ON
 
+# External library: libcxxwrap-julia
+if(openPMD_USE_JULIA STREQUAL AUTO)
+    find_package(JlCxx 0.8.3)
```

Comment:
```suggestion
    find_package(JlCxx 0.8.3 QUIET)
```

`find_package` gives some very verbose output for JlCxx at least for me:
```
CMake Warning at CMakeLists.txt:327 (find_package):                                             
  By not providing "FindJlCxx.cmake" in CMAKE_MODULE_PATH this project has                      
  asked CMake to find a package configuration file provided by "JlCxx", but                     
  CMake did not find one.                                                                       
                                                                                                
  Could not find a package configuration file provided by "JlCxx" (requested                    
  version 0.8.3) with any of the following names:                                               
                                                                                                
    JlCxxConfig.cmake                                                                           
    jlcxx-config.cmake                                                                          
                                                                                                
  Add the installation prefix of "JlCxx" to CMAKE_PREFIX_PATH or set                            
  "JlCxx_DIR" to a directory containing one of the above files.  If "JlCxx"                     
  provides a separate development package or SDK, be sure it has been                           
  installed.
```
I don't think that there should be a warning if `openPMD_USE_JULIA` is on `AUTO`? Maybe prefer something like:
```cmake
message(STATUS "Found JlCxx in ${JlCxx_location}")
```

Reply:
Downside of this is that you would lose the very detailed output that you get if JlCxx is actually found:
```
-- Found Julia executable: /home/franzpoeschel/spack/openpmd-dev/.spack-env/view/bin/julia
-- Julia_VERSION_STRING: 1.6.2
-- Julia_INCLUDE_DIRS:   /opt/spack-modules/linux-ubuntu20.04-skylake_avx512/gcc-10.2.0/julia-1.6.2-3zh2z2gcuy2n3phg4pokwvoz5oxsacif/include/julia
-- Julia_LIBRARY_DIR:    /opt/spack-modules/linux-ubuntu20.04-skylake_avx512/gcc-10.2.0/julia-1.6.2-3zh2z2gcuy2n3phg4pokwvoz5oxsacif/lib
-- Julia_LIBRARY:        /opt/spack-modules/linux-ubuntu20.04-skylake_avx512/gcc-10.2.0/julia-1.6.2-3zh2z2gcuy2n3phg4pokwvoz5oxsacif/lib/libjulia.so.1
-- JULIA_HOME:           /opt/spack-modules/linux-ubuntu20.04-skylake_avx512/gcc-10.2.0/julia-1.6.2-3zh2z2gcuy2n3phg4pokwvoz5oxsacif/bin
-- Julia_LLVM_VERSION:   v11.0.1
-- Julia_WORD_SIZE:      64
-- Found Julia: /opt/spack-modules/linux-ubuntu20.04-skylake_avx512/gcc-10.2.0/julia-1.6.2-3zh2z2gcuy2n3phg4pokwvoz5oxsacif/lib/libjulia.so.1 (found version "1.6.2") 
-- Found JlCxx version 0.8.3 at /home/franzpoeschel/spack/openpmd-dev/.spack-env/view/lib
```
But if you're interested, one can argue that you will use `openPMD_USE_JULIA=ON` anyway?

Reply:
Most people will install the Julia bindings via the Julia package manager. I think that `USE_JULIA` should default to `OFF`, not `AUTO`. Would that help? This way, the verbose output would only be presented when people explicitly ask for it.

Reply:
That's a good idea and would also simplify the find logic here.